pkg/services/alerting/notifier_test.go

```diff
@@ -18,18 +19,19 @@ import (
 )
 
 func TestNotificationService(t *testing.T) {
-	testRule := &Rule{
-		ID:            1,
-		DashboardID:   1,
-		PanelID:       1,
-		OrgID:         1,
-		Name:          "Test",
-		Message:       "Something is bad",
-		State:         models.AlertStateAlerting,
-		Notifications: []string{"1"},
-	}
+	testRule := newTestRule("Test", "Something is bad")
 	evalCtx := NewEvalContext(context.Background(), testRule)
 
+	testRuleTemplated := newTestRule("Test {{.quantile}}", "Something is bad on instance {{ .instance }}")
+	evalCtxWithMatch := newEvalContext(context.Background(), testRuleTemplated, &EvalMatch{
```

Comment:
Can you just inline newEvalContext? Since it's only used once and super simple, I don't see any benefit to it. It just causes more complexity (especially with the condition) than inlining.